client/src/main/java/io/confluent/kafka/schemaregistry/client/SchemaRegistryClientConfig.java

```diff
@@ -21,6 +21,8 @@
   @Deprecated
   public static final String SCHEMA_REGISTRY_USER_INFO_CONFIG =
       "schema.registry.basic.auth.user.info";
-  public static final String USER_INFO_CONFIG =
-      "basic.auth.user.info";
+  public static final String USER_INFO_CONFIG = "basic.auth.user.info";
```

Comment:
We need to add these configs to `AbstractKafkaAvroSerDeConfig` as well do we not?

Reply:
fixed